_includes/manuals/3.17/1.2_release_notes.md

```diff
@@ -4,16 +4,230 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### PatternFly 5 UI Updates
+Continued migration to PatternFly 5 brings modernized notifications ([#38345](https://projects.theforeman.org/issues/38345)) and other UI components, improving the overall user experience with updated design patterns.
+
+#### Multi-Architecture iPXE Support
+iPXE chain UEFI now supports multiple architectures ([#38300](https://projects.theforeman.org/issues/38300)), enabling proper provisioning of ARM64 and other non-x86_64 servers by delivering the correct iPXE binary for each architecture.
+
+#### Ubuntu 24 Support
+Added support for Ubuntu 24 as a supported platform ([#38682](https://projects.theforeman.org/issues/38682)).
```

Comment:
```suggestion
Added Ubuntu 24 as a supported platform for Foreman installer ([#38682](https://projects.theforeman.org/issues/38682)).
```

Reply:
I don't see the option to install Foreman on Ubuntu 24 documented in the [Quickstart](https://docs.theforeman.org/3.17/Quickstart/index-foreman-deb.html).

Reply:
I've removed this item from Release Notes.